editor/dependency_editor.cpp

```diff
@@ -495,6 +495,10 @@ void DependencyRemoveDialog::ok_pressed() {
 			Resource *res = ResourceCache::get(files_to_delete[i]);
 			res->set_path("");
 		}
+      // If the file we are deleting is the main scene, clear the info for the main scene in project.godot.
+      if (files_to_delete[i] == ProjectSettings::get_singleton()->get("application/run/main_scene")) {
+         ProjectSettings::get_singleton()->set("application/run/main_scene", "");
+      }
```

Comment:
You should be using tabs to align, not spaces.

Reply:
I forgot that I had my IDE set to use spaces. I'm re-submitting now.